src/classes/Client.ts

```diff
@@ -23,23 +16,24 @@ import TagHandler from './handlers/TagHandler';
 import PhishingHandler from '@handlers/PhishingHandler';
 
 export default class FluorineClient extends Client {
-    applicationCommands!: ApplicationCommands;
-    conn!: r.Connection;
-    cmds!: Collection<string, Command>;
-    components!: Collection<string, Component>;
-    economy: EconomyHandler;
-    phishing: PhishingHandler;
-    shop: ShopHandler;
-    tags: TagHandler;
-    cooldown: Set<string>;
-    ai: AI;
-    invite: string;
-    version: string;
-    color: ColorResolvable;
-    devs: string[];
+    logger = Logger;
+
+    applicationCommands = new ApplicationCommandHandler(this);
+    conn: r.Connection;
+    cmds = new CommandHandler(this).loadCommands();
+    components = new ComponentHandler(this);
+    economy = new EconomyHandler(this);
+    phishing = new PhishingHandler(this);
+    shop = new ShopHandler(this);
+    tags = new TagHandler(this);
+    cooldown = new Set<string>();
+    ai = new AI(this);
+    invite =
+        'https://discord.com/api/oauth2/authorize?client_id=831932409943425064&scope=bot+applications.commands&permissions=474527689975';
+    version = process.env.npm_package_version;
+    devs = ['707675871355600967', '478823932913516544', '348591272476540928'];
     generating: boolean;
```

Comment:
this is useless i think
![image](https://user-images.githubusercontent.com/32780515/155684816-8f934d5f-67d9-4741-92f0-dddeb7567038.png)

Reply:
@xiboon you are the one who added this what do you think

Reply:
im pretty sure this was moved to client.ai.generating

Reply:
yep

its isGenerating but ok 👍